Dockerfile

```diff
@@ -25,35 +26,80 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
     GOOS=$TARGETOS GOARCH=$TARGETARCH \
     go build -ldflags "-X github.com/treeverse/lakefs/pkg/version.Version=${VERSION}" -o lakectl ./cmd/lakectl
 
+# Build delta diff binary
+FROM --platform=$BUILDPLATFORM rust:1.68-bullseye AS build-delta-diff-plugin
+RUN USER=root cargo new --bin delta-diff
+WORKDIR /delta-diff
+
+# 2. Copy our manifests
+COPY ./pkg/plugins/diff/delta_diff_server/Cargo.lock ./Cargo.lock
+COPY ./pkg/plugins/diff/delta_diff_server/Cargo.toml ./Cargo.toml
```

Comment:
Will this work?  It will reduce one level of the image, which is sort-of convenient:
```suggestion
COPY ./pkg/plugins/diff/delta_diff_server/Cargo.* ./
```